### PR TITLE
move to ServiceBus IAM

### PR DESCRIPTION
**Pull request by vbelinschi**
## References

* #0000

### CODE REVIEW

File: source/marketparticipant/Energinet.DataHub.MarketParticipant.EntryPoint.Organization/Extensions/DependencyInjection/MarketParticipantOrganizationModuleExtensions.cs [hunk 1]
```diff
@@ -19,17 +19,18 @@
...
 using Energinet.DataHub.MarketParticipant.Application.Options;
 using Energinet.DataHub.MarketParticipant.Application.Services;
 using Energinet.DataHub.MarketParticipant.Common;
 using Energinet.DataHub.MarketParticipant.EntryPoint.Organization.Functions;
 using Energinet.DataHub.MarketParticipant.EntryPoint.Organization.Integration;
 using Energinet.DataHub.MarketParticipant.EntryPoint.Organization.Monitor;
-using Energinet.DataHub.MarketParticipant.EntryPoint.Organization.Options;
 using Energinet.DataHub.MarketParticipant.Infrastructure.Persistence;
 using Energinet.DataHub.MarketParticipant.Infrastructure.Persistence.Repositories;
+using Google.Api;
```
Comment (defectiveAi) [inline source/marketparticipant/Energinet.DataHub.MarketParticipant.EntryPoint.Organization/Extensions/DependencyInjection/MarketParticipantOrganizationModuleExtensions.cs:33]: ?

File: source/marketparticipant/Energinet.DataHub.MarketParticipant.IntegrationTests/Fixtures/WebApiIntegrationTestsBase.cs [hunk 2]
```diff
@@ -80,10 +80,8 @@ protected override void ConfigureWebHost(IWebHostBuilder builder)
         builder.UseSetting("KeyVault:TokenSignKeyVault", "https://fake_value");
         builder.UseSetting("KeyVault:TokenSignKeyName", "fake_value");
         builder.UseSetting("KeyVault:CertificatesKeyVault", "https://fake_value");
-        builder.UseSetting("ServiceBusOptions:SharedIntegrationEventTopic", "fake_value");
-        builder.UseSetting("ServiceBusOptions:IntegrationEventSubscription", "fake_value");
-        builder.UseSetting("ServiceBusOptions:ProducerConnectionString", "fake_value");
-        builder.UseSetting("ServiceBusOptions:HealthConnectionString", "fake_value");
+        builder.UseSetting($"{IntegrationEventsOptions.SectionName}:{nameof(IntegrationEventsOptions.TopicName)}", "fake_value");
```
Comment (defectiveAi) [inline source/marketparticipant/Energinet.DataHub.MarketParticipant.IntegrationTests/Fixtures/WebApiIntegrationTestsBase.cs:83]: WebApi does not talk to ServiceBus, would if you can just delete both of these registrations.

File: source/marketparticipant/Energinet.DataHub.MarketParticipant.IntegrationTests/OrganizationIntegrationTestHost.cs [hunk 3]
```diff
@@ -76,10 +77,8 @@ private static IConfiguration BuildConfig(string dbConnectionString)
 
             new("CvrUpdate:NotificationToEmail", "fake_value@fake_value_test.dk"),
             new("BalanceResponsibleChanged:NotificationToEmail", "fake_value@fake_value_test.dk"),
-            new("ServiceBus:SharedIntegrationEventTopic", "fake_value"),
-            new("ServiceBus:IntegrationEventSubscription", "fake_value"),
-            new("ServiceBus:ConsumerConnectionString", "fake_value"),
-            new("ServiceBus:ProducerConnectionString", "fake_value"),
+            new($"{IntegrationEventsOptions.SectionName}:{nameof(IntegrationEventsOptions.TopicName)}", "fake_value"),
+            new($"{IntegrationEventsOptions.SectionName}:{nameof(IntegrationEventsOptions.SubscriptionName)}", "fake_value"),
```
Comment (defectiveAi) [inline source/marketparticipant/Energinet.DataHub.MarketParticipant.IntegrationTests/OrganizationIntegrationTestHost.cs:81]: Suspicious that FullyQualifiedNamespace is not needed to be provided.

File: source/marketparticipant/Energinet.DataHub.MarketParticipant.IntegrationTests/WebApiIntegrationTestHost.cs [hunk 4]
```diff
@@ -93,10 +94,8 @@ private static IConfiguration BuildConfig(string dbConnectionString)
             new("AzureB2c:BackendObjectId", "fake_value"),
             new("AzureB2c:BackendSpnObjectId", "fake_value"),
             new("AzureB2c:BackendId", "fake_value"),
-            new("ServiceBusOptions:SharedIntegrationEventTopic", "fake_value"),
-            new("ServiceBusOptions:IntegrationEventSubscription", "fake_value"),
-            new("ServiceBusOptions:ProducerConnectionString", "fake_value"),
-            new("ServiceBusOptions:HealthConnectionString", "fake_value"),
+            new($"{IntegrationEventsOptions.SectionName}:{nameof(IntegrationEventsOptions.TopicName)}", "fake_value"),
```
Comment (defectiveAi) [inline source/marketparticipant/Energinet.DataHub.MarketParticipant.IntegrationTests/WebApiIntegrationTestHost.cs:97]: Think both lines can be removed.

File: source/marketparticipant/Energinet.DataHub.MarketParticipant.EntryPoint.Organization/Extensions/DependencyInjection/MarketParticipantOrganizationModuleExtensions.cs (hunk 1 above)
Comment (vbelinschi) [reply source/marketparticipant/Energinet.DataHub.MarketParticipant.EntryPoint.Organization/Extensions/DependencyInjection/MarketParticipantOrganizationModuleExtensions.cs]: yea, wtf? 😄 
someking of autocomplete

File: source/marketparticipant/Energinet.DataHub.MarketParticipant.IntegrationTests/Fixtures/WebApiIntegrationTestsBase.cs (hunk 2 above)
Comment (vbelinschi) [reply source/marketparticipant/Energinet.DataHub.MarketParticipant.IntegrationTests/Fixtures/WebApiIntegrationTestsBase.cs]: it works

File: source/marketparticipant/Energinet.DataHub.MarketParticipant.IntegrationTests/OrganizationIntegrationTestHost.cs (hunk 3 above)
Comment (vbelinschi) [reply source/marketparticipant/Energinet.DataHub.MarketParticipant.IntegrationTests/OrganizationIntegrationTestHost.cs:81]: I am not sure why, but if I remove these two lines it fails as expected

**Review by defectiveAi — APPROVED** [review]

File: source/marketparticipant/Energinet.DataHub.MarketParticipant.IntegrationTests/OrganizationIntegrationTestHost.cs (hunk 3 above)
Comment (vbelinschi) [reply source/marketparticipant/Energinet.DataHub.MarketParticipant.IntegrationTests/OrganizationIntegrationTestHost.cs:82]: you were right, it is needed
with the new updated Messaging package, the registration of the publisher was changed, and I didn't do it right the first time

**Review by michaelmajgaard — APPROVED** [review]

**Pull request merged**